modular_skyrat/master_files/code/modules/power/lighting.dm

```diff
@@ -0,0 +1,6 @@
+//We're lowering the power of light fixtures for **mood lighting**
```

Comment:
Please ensure you add an entry into the readme.md in the masterfiles folder, to be updated soon.